platforms/web/vite.config.ts

```diff
@@ -34,6 +34,7 @@ export default defineConfig({
                 'lib/types.ts',
                 'lib/constants.ts',
                 'lib/useListeners/types.ts',
+                'lib/useTestCases/types.ts',
```

Comment:
Hmmm, why do we need to export a type used in test cases?

Reply:
Yea, its a bit weird that the function in question I think just returns this type so that we can unit test it.

At a high level you are right though, the app code shouldn't depend on the test code. I'll move the type to the app code to switch this around.